<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(119, 118, 123);</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>58</y>
     <width>252</width>
     <height>230</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_4" rowstretch="0,0,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetFixedSize</enum>
    </property>
    <property name="horizontalSpacing">
     <number>5</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QPushButton" name="push_7">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 400 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>7</string>
      </property>
      <property name="shortcut">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="push_1">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 400 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="shortcut">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="push_5">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 400 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>5</string>
      </property>
      <property name="shortcut">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="push_4">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 400 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>4</string>
      </property>
      <property name="shortcut">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="push_2">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 400 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>2</string>
      </property>
      <property name="shortcut">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="push_6">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 400 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>6</string>
      </property>
      <property name="shortcut">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="push_8">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 400 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>8</string>
      </property>
      <property name="shortcut">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="push_9">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 400 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>9</string>
      </property>
      <property name="shortcut">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="push_3">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 400 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>3</string>
      </property>
      <property name="shortcut">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="push_dot">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 13pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>.</string>
      </property>
      <property name="shortcut">
       <string>.</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="push_0">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 400 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="shortcut">
       <string>0</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="push_c">
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 600 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>C</string>
      </property>
      <property name="shortcut">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>51</y>
     <width>471</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <item row="2" column="3">
     <widget class="QLineEdit" name="gcv">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="fixCharg">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLineEdit" name="curReading">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="maxLength">
       <number>8</number>
      </property>
      <property name="placeholderText">
       <string>00000000</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLabel" name="meterRentLabel">
      <property name="styleSheet">
       <string notr="true">font: 400 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Meter Rent</string>
      </property>
     </widget>
    </item>
    <item row="4" column="3">
     <widget class="QLineEdit" name="meterRent">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="currentLabel">
      <property name="styleSheet">
       <string notr="true">font: 400 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Current</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="factorLabel">
      <property name="styleSheet">
       <string notr="true">font: 400 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Factor</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="previousLabel">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 400 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Previous</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="gCVLabel">
      <property name="styleSheet">
       <string notr="true">font: 400 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>GCV</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="factor">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="fixChargesLabel">
      <property name="styleSheet">
       <string notr="true">font: 400 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>Fix Charges</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="preReading">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
</string>
      </property>
      <property name="inputMethodHints">
       <set>Qt::InputMethodHint::ImhNone</set>
      </property>
      <property name="inputMask">
       <string/>
      </property>
      <property name="maxLength">
       <number>8</number>
      </property>
      <property name="placeholderText">
       <string>00000000</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="footer">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>12</y>
     <width>267</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>267</width>
     <height>16777202</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Gas Bill Calculator (SNGPL)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushCalculate">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>471</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 600 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QLabel" name="header">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>361</y>
     <width>327</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777202</width>
     <height>16777202</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;Sui Northern Gas Pipelines Limited (SNGPL)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="totAmount">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>471</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(246, 245, 244);</string>
   </property>
   <property name="placeholderText">
    <string>Amount</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushOnlineBill">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>300</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 400 12pt &quot;Arial&quot;;
color: rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>Online Bill</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
